app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1078,13 +1079,8 @@ BrowserWorld::StartFrame() {
     }
   }
 
-  // @FIXME: Make this a state variable and toggle it with the menu item
-  //         Also replace the #if below with a proper SupportsPassthrough()
-  //         method based on build type.
-  bool passthroughEnabled = false;
-
 #if defined(OCULUSVR)
```

Comment:
We should get rid of this #ifdef, we're already filtering on Java side with `isPassthroughSupported`. Also this is a bit inconsistent as we're enabling passthrough for Oculus&Pico and here the ifdef is only allowing Oculus.

Reply:
I removed the Pico part of that ifdef because Pico support is not working yet anyway. When I find out what's missing in Android Manifest and am able to test passthrough on Pico, I will send a patch to add support.

In other words, current support is only for Oculus at the end, so this should be fine.

Reply:
ok but we can remove he ifdef anyway, and just make the boolean true with `BuildConfig.isOculusBuild()`

Reply:
It won't hurt but as we only have 1 code path arriving there... not sure it pays off. I'll let you decide

Reply:
Umm, right. It'd only be useful if we were able to have a reliable check for the OpenXR extension support.